### PR TITLE
Fix !mp make bug by refactoring createMatch to return full obj

### PR DESCRIPTION
**Pull request by cmyui**
Made this for the purpose of fixing the following bug:

image

I went with a slightly-more-than-minimal diff as I think this simplifies the overall code

### CODE REVIEW

File: constants/chatbotCommands.py [hunk 1]
```diff
@@ -1761,11 +1761,9 @@ async def mpMake() -> Optional[str]:
             host_user_id=-1,
             is_tourney=True,
         )
-        multiplayer_match = await matchList.getMatchFromChannel(chan)
```
Comment (cmyui) [inline constants/chatbotCommands.py:1764]: @lenforiee - the error you noticed was happening in this function call -- `chan` is the channel from which the player is running the command -- which will never be the right `#mp_{id}` channel.

down the call chain this results in the raising of `wrongChannelException`, resulting in the response msg you observed _after_ match creation from exc handling in the higher level `multiplayer()` command handler

File: constants/chatbotCommands.py [hunk 2]
```diff
@@ -2446,16 +2444,22 @@ async def mpSettings() -> Optional[str]:
...
-                    else "!! no team !!",
+                    team=(
+                        "red"
+                        if _slot["team"] == matchTeams.RED
+                        else (
+                            "blue"
+                            if _slot["team"] == matchTeams.BLUE
+                            else "!! no team !!"
+                        )
+                    ),
```
Comment (cmyui) [inline constants/chatbotCommands.py]:
```suggestion
                    team="red"
                    if _slot["team"] == matchTeams.RED
                    else "blue"
                    if _slot["team"] == matchTeams.BLUE
                    else "!! no team !!",
```

File: constants/chatbotCommands.py [hunk 3]
```diff
@@ -2446,16 +2444,22 @@ async def mpSettings() -> Optional[str]:
...
                     status=readableStatus,
                     username=slot_token["username"],
-                    mods=f" (+ {scoreUtils.readableMods(_slot['mods'])})"
-                    if _slot["mods"] > 0
-                    else "",
+                    mods=(
+                        f" (+ {scoreUtils.readableMods(_slot['mods'])})"
+                        if _slot["mods"] > 0
+                        else ""
+                    ),
```
Comment (cmyui) [inline constants/chatbotCommands.py]:
```suggestion
                    mods=f" (+ {scoreUtils.readableMods(_slot['mods'])})"
                    if _slot["mods"] > 0
                    else "",
```

**Comment by cmyui** [comment]
Untested -- I'm on a macbook atm and won't be able to test until next week.

Will not merge until tested.

**Comment by tsunyoku** [comment]
superceded by https://github.com/osuAkatsuki/bancho-service/pull/112

**Pull request closed without merging**